Library/Homebrew/cmd/update-bash.sh

```diff
@@ -305,7 +323,7 @@ EOS
   wait
   trap - SIGINT
 
-  for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
+  for DIR in "${DIRS[@]}"
   do
     [[ -d "$DIR/.git" ]] || continue
     pull "$DIR"
```

Comment:
I think we should skip pull as well. Current pull won't work with case of init git in the first time. So instead of patching pull function, why not just skip it.